backend/src/v1/services/report-service.ts

```diff
@@ -664,20 +698,51 @@ const reportService = {
       explanatoryNotes: this.createExplanatoryNotes(report),
     };
 
-    const responseHtml= await utils.postDataToDocGenService(
+    const responseHtml = await utils.postDataToDocGenService(
       reportData,
       `${config.get('docGenService:url')}/doc-gen?reportType=html`,
       req.session.correlationID,
     );
-    logger.debug(`getReportHtml completed with reportId: ${reportId} and correlationId: ${req.session?.correlationID}`);
+    logger.debug(
+      `getReportHtml completed with reportId: ${reportId} and correlationId: ${req.session?.correlationID}`,
+    );
     return responseHtml;
   },
+
+  /**  Return all published reports created by a company */
+  async getReportsByStatus(
+    bceidBusinessGuid: string,
+    status: enumReportStatus,
+  ) {
+    const reports = await prisma.pay_transparency_company.findFirst({
+      select: {
+        pay_transparency_report: {
```

Comment:
Can we sort the reports by report start date (newest to oldest), then secondarily by update date (newest to oldest)?